spk/domoticz/src/installer.sh

```diff
@@ -75,6 +77,16 @@ preupgrade ()
     # Save some stuff
     rm -fr ${TMP_DIR}/${PACKAGE}
     mkdir -p ${TMP_DIR}/${PACKAGE}
+
+    # Revision 7 moves  scripts, backups and other user files to ${INSTALL_DIR}/var/
+    if [ `echo ${SYNOPKG_OLD_PKGVER} | sed -r "s/^.*-([0-9]+)$/\1/"` -le 6 ]; then
+    # backups dir is optional, but below move silently fails if directory doesn't exist.
+    mv ${INSTALL_DIR}/backups ${INSTALL_DIR}/var/
+    mv ${INSTALL_DIR}/scripts ${INSTALL_DIR}/var/
+    mv ${INSTALL_DIR}/Config/options.xml  ${INSTALL_DIR}/var/
+    mv ${INSTALL_DIR}/Config/zwcfg*.xml  ${INSTALL_DIR}/var/
+    fi
+
```

Comment:
OK we can still do that later on so let's keep it for now but please fix the indentation inside the if block.